tests/scripts/analyze_outcomes.py

```diff
@@ -207,6 +207,24 @@ def do_analyze_driver_vs_reference(outcome_file, args):
             }
         }
     },
+    'analyze_driver_vs_reference_all_ec_algs': {
```

Comment:
Shouldn't removed components be also removed from here?

Reply:
Ops, yes, of course, you are totally right.